### PR TITLE
[PagoPA-922] update onus logic

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- changed the logic of the onus valorisation 

#### Motivation and Context

- adaptation to new valorisation rules

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

File: src/main/java/it/gov/pagopa/afm/calculator/exception/AppError.java [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 @Getter
 public enum AppError {
-  ISSUERS_NOT_FOUND(
-      HttpStatus.NOT_FOUND, "Issuers not found", "Not found any issuer for the bin %s"),
+  /*ISSUERS_NOT_FOUND(
+      HttpStatus.NOT_FOUND, "Issuers not found", "Not found any issuer for the bin %s"),*/
```
Comment (FedericoRuzzier) [inline src/main/java/it/gov/pagopa/afm/calculator/exception/AppError.java:9]: Delete comment

**Review by FedericoRuzzier — APPROVED** [review]
Delete comment then LGTM

File: src/main/java/it/gov/pagopa/afm/calculator/exception/AppError.java (hunk 1 above)
Comment (alessio-acitelli) [reply src/main/java/it/gov/pagopa/afm/calculator/exception/AppError.java]: done

**Comment by alessio-acitelli** [comment]
update_code

**Pull request merged**